transformer_engine/pytorch/utils.py

```diff
@@ -179,8 +179,16 @@ def cast_if_needed(tensor: torch.Tensor, dtype: torch.dtype) -> torch.Tensor:
         return tensor if tensor is None or tensor.dtype == dtype else tensor.to(dtype)
 
 
-def check_dim_for_fp8_forward_exec(*tensors: Tuple[torch.Tensor, ...]) -> bool:
+def check_dim_for_fp8_forward_exec(*tensors: torch.Tensor) -> bool:
     """For fp8 fprop (TN layout), inputs and weights must be such
        that dim0 is divisible by 8 and dim1 is divisible by 16.
     """
     return all(not t.shape[0] % 8 and not t.shape[1] % 16 for t in tensors)
+
+
+def assert_dim_for_fp8_forward_exec(tensor: torch.Tensor) -> None:
```

Comment:
The linter is complaining about a missing docstring:
```suggestion
def assert_dim_for_fp8_forward_exec(tensor: torch.Tensor) -> None:
    """For fp8 fprop (TN layout), inputs and weights must be such
       that dim0 is divisible by 8 and dim1 is divisible by 16.
    """
```

Reply:
Thanks for the review! Incorporated